shared/js/background/utils.es6.js

```diff
@@ -114,6 +114,10 @@ function getAsyncBlockingSupport () {
     return false
 }
 
+function getPersistentBackgroundPageSupport () {
+    return (typeof chrome.runtime.getBackgroundPage) === 'function'
```

Comment:
In MV3 there is no access to background page through getBackgroundPage, we need to send messages.